cmd/entrypoint/webhooks.go

```diff
@@ -0,0 +1,275 @@
+package entrypoint
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/rand"
+	"crypto/tls"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"math/big"
+	"net/http"
+	"context"
+	"time"
+	b "bytes"
+	"fmt"
+
+	"github.com/datawire/ambassador/v2/pkg/k8s"
```

Comment:
Prefer to use pkg/kates instead of pkg/k8s

Reply:
k8s here is only used for NewKubeInfo() which kates does not seem to replicate.